client/packages/common/src/types/schema.ts

```diff
@@ -633,7 +633,6 @@ export type DatabaseError = DeleteLocationErrorInterface & InsertLocationErrorIn
 
 export type DatabaseSettingsNode = {
   __typename: 'DatabaseSettingsNode';
-  /** Central server url */
```

Comment:
Not sure how this happened... maybe codegen didn't run since merging in develop?